{
	"auth.error.incorrect_login_or_password": "Не верно указан логин или пароль.",
	"auth.error.not_auth_or_token_expired": "Вы не авторизованы или время жизни токена сессии подошло к концу.",
	"auth.error.you_already_auth": "Вы уже авторизированы.",
	"auth.error.creating_user_details": "Пользователь не создался в базе данных в таблице UserDetails.",
	"auth.error.creating_user": "Пользователь не создался в базе данных в таблице Users.",
	"auth.error.session_id_not_exists": "Уникальный идентификатор сессии не существует.",

	"auth.error.user_with_email_already_exists": "Пользователь с почтовым адресом {{email}} уже существует.",
	"auth.error.user_with_phone_already_exists": "Пользователь с номером телефона {{phone}} уже существует.",
	"auth.error.session_id_not_exists_on_deleted_session": "Отсутствует идентификатор сессии пользователя (session={{session}}).",
	"auth.error.user_not_exists": "Пользователь не существует.",

	"photos.error.user_not_found": "Пользователь не найден.",
	"photos.error.sharp_avatar_path_not_found": "Не передан путь к сжатому аватару.",
	"photos.error.sharp_photo_path_not_found": "Не передан путь к сжатой фотографии.",
	"photos.error.sharp_photo_paths_not_found": "Сжатые фотографии не найдены.",
	"photos.error.delete_photo_path_not_found": "Не передан путь для удаления фотографии.",
	"photos.error.photo_not_found": "Изображение не найдено.",
	"photos.error.photos_not_found": "Изображения не найдены.",
	"photos.error.sharped_photo_with_error": "Возникла ошибка при сжатии изображений: {{errorMessage}}",

	"files.error.message_id_not_found": "Не передан id сообщения.",
	"files.error.files_not_found": "Не переданы файлы.",
	"files.error.file_path_open_not_found": "Не передан путь для открытия файла.",
	"files.error.file_path_download_not_found": "Не передан путь для скачивания файла.",
	"files.error.file_not_found": "Файл не найден.",

	"friends.error.get_friends_count": "Возникла ошибка при получении количества запросов в друзья.",
	"friends.error.get_possible_friends": "Возникла ошибка при получении рекомендуемых друзей.",
	"friends.error.get_all_friends": "Возникла ошибка при получении всех друзей.",
	"friends.error.get_friends_online": "Возникла ошибка при получении друзей онлайн.",
	"friends.error.get_subscribers": "Возникла ошибка при получении подписчиков.",
	"friends.error.get_incoming_requests": "Возникла ошибка при получении входящих заявок на дружбу.",
	"friends.error.get_outgoing_requests": "Возникла ошибка при получении исходящих заявок на дружбу.",
	"friends.error.search_friends": "Возникла ошибка при поиске возможных друзей.",
	"friends.error.unknown_type_friends": "Не распознан тип получаемых друзей.",

	"friends.error.friend_id_not_found": "Информация о пользователе не найдена, возможно, это временный чат.",
	"friends.error.subscribed_id_not_found": "Не передан идентификатор пользователя, на которого происходит подписка.",
	"friends.error.unsubscribed_id_not_found": "Не передан идентификатор пользователя, от которого происходит отписка.",
	"friends.error.added_id_not_found": "Не передан идентификатор пользователя, которого принимаем в друзья.",
	"friends.error.left_to_subscribed_id_not_found": "Не передан идентификатор пользователя, которого оставляем в подписчиках.",
	"friends.error.deleted_id_not_found": "Не передан идентификатор пользователя, которого удаляем из друзей.",
	"friends.error.blocked_id_not_found": "Не передан идентификатор пользователя, которого блокируем.",
	"friends.error.check_blocked_id_not_found": "Не передан идентификатор пользователя при проверке на его блокировку.",

	"users.error.user_not_found": "Пользователь не найден.",
	"users.error.user_details_not_found": "Дополнительная информация о пользователе не найдена.",
	"users.error.user_id_not_found": "Идентификатор пользователя не передан.",
	"users.error.user_with_id_not_found": "Пользователь с id={{id}} не найден.",
	"users.error.user_incorrect_data": "Некорректные данные пользователя.",

	"messages.error.get_unread_messages": "Возникла ошибка при получении непрочитанных сообщений.",
	"messages.error.user_id_in_message_not_found": "Не передан Ваш уникальный идентификатор в объекте сообщения.",
	"messages.error.chat_id_in_message_not_found": "Не передан уникальный идентификатор чата в объекте сообщения.",
	"messages.error.message_id_not_found": "Не передан уникальный идентификатор сообщения.",
	"messages.error.empty_edit_message_without_files": "Ошибка при изменении сообщения. Оно не может быть пустым без прикрепленных файлов.",
	"messages.error.deleted_message_id_not_found": "Не передан уникальный идентификатор удаляемого сообщения.",
	"messages.error.your_already_delete_this_message": "Данное сообщение уже было Вами удалено.",

	"chats.error.chat_id_not_found": "Не передан уникальный идентификатор чата.",
	"chats.error.partner_id_not_found": "Не передан уникальный идентификатор собеседника.",
	"chats.error.partner_not_found": "Собеседник чата не найден.",
	"chats.error.deleted_chat_id_not_found": "Не передан уникальный идентификатор удаляемого приватного чата.",
	"chats.error.users_in_chat_not_found": "Пользователи в чате {{chat}} не найдены.",
	"chats.error.chat_not_found": "Чат с идентификатором {{chatId}} не найден.",

	"socket.disconnecting": "В процессе отключения сокет соединения ({{reason}}).",
	"socket.socket_disconnected_with_reason": "Сокет с идентификатором соединения {{socketId}} отключился по причине: {{reason}}.",
	"socket.emit_handled": "[{{timestamp}}] | Событие {{type}} обработано на клиенте.",
	"socket.handshake_connection_successful": "Сокет-соединение готово для установки нового соединения.",
	"socket.session_done": "Проверка сессии в хранилище Redis успешно завершена.",

    "socket.error.user_with_id_not_exists": "Пользователя с идентификатором {{userId}} не существует.",
    "socket.error.user_id_not_found": "Не передан идентификатор пользователя.",
	"socket.error.user_not_found": "Пользователь не существует.",
    "socket.error.not_correct_friends_action_type": "Некорректный тип действия при работе с друзьями.",
    "socket.error.cannot_find_call": "Невозможно завершить звонок, так как он не найден. Возможно, он уже завершен.",
    "socket.error.not_correct_user_id_in_socket": "Не подходящий идентификатор пользователя id={{userTo}}. Необходимо использовать метод notifyMe.",
    "socket.error.disconnect": "Возникла ошибка при отключении пользователя: {{message}}.",
    "socket.error.socket_work": "Произошла ошибка при работе с сокетом",
	"socket.error.emit_with_ack": "Произошла ошибка при попытке отправки события {{type}}: {{message}}",
	"socket.error.emit_broadcast_with_ack": "Произошла ошибка при попытке отправки всем пользователям, кроме себя, события {{type}}: {{message}}",
	"socket.error.emit_to_with_ack": "Произошла ошибка при попытке отправки события {{type}} конкретному пользователю: {{message}}",
	"socket.error.emit_broadcast_empty_response": "Сбой в работе сокет-соединения. Список подтвержденных обработок событий от клиентов пуст.",
	"socket.error.handle_event_on_client": "При обработке данного события {{type}} на клиенте возникла ошибка: {{message}}",
	"socket.error.sessions_not_match_or_exists": "При проверке сессии пользователя [sessionId={{sessionId}}] возникла ошибка. Вы не авторизованы или время жизни токена сессии подошло к концу.",
    "socket.error.engine_socket": "Не нормальное отключение сокета с кодом {{code}}.\nТекст ошибки: {{message}}.\nДополнительный контекст: {{context}}.",
	"socket.error.validation_emit": "При попытке отправить событие клиенту произошла ошибка валидации схемы: {{message}}",

	"database.start": "Соединение с базой данных успешно установлено.",
	"database.close": "Соединение с бд успешно закрыто.",
	"database.error.close": "Закрытие соединения с базой данных завершилось не удачно.",
	"database.error.connect": "Соединение с базой данных завершилось с ошибкой.",

    "redis.connection_successfull": "Соединение с клиентом Redis успешно установлено.",
    "redis.start_to_work": "Клиент Redis готов к работе.",
    "redis.stopped": "Клиент Redis остановлен.",
    "redis.reconnection": "Клиент Redis переподключается...",
    "redis.ttl_update": "Время жизни в размере {{ttl}} для ключа {{key}} успешно обновлено в Redis.",
    "redis.new_pair_is_set": "Новая пара [{{key}}:{{value}}] успешно записана в Redis.",
    "redis.key_successfull_deleted": "Ключ [{{key}}] успешно удален из Redis.",

    "redis.error.client_connect": "Соединение с базой данных Redis завершилось не удачно: ",
    "redis.error.client_work": "Ошибка в работе клиента Redis: ",
    "redis.error.ttl_update": "Произошла ошибка при обновлении времени жизни записи с ключом [{{key}}] из Redis: {{message}}.",
    "redis.error.get_value": "Произошла ошибка при получении значения по ключу [{{key}}] из Redis.",
    "redis.error.setting_new_pair": "Произошла ошибка при записи новой пары [{{key}}:{{value}}] в Redis.",
    "redis.error.deleted_key": "Произошла ошибка при удалении ключа [{{key}}] из Redis.",

	"server_started": "Экземпляр сервера запущен на порту: {{port}} в режиме: {{mode}}.",
	"start_server_error": "Возникла ошибка при запуске экземпляра сервера",

	"error.unhandled_sync": "Не обработанная ошибка в синхронном коде: {{errorMessage}}",
	"error.unhandled_async": "Не обработанная ошибка в асинхронном коде: {{reason}} ({{promise}})",
	"error.initialize_i18n": "Возникла ошибка при инициализации пакета i18n.",

	"validation.socket.emit_event": "Валидация схемы при отправке события {{event}}.",
	"validation.socket.handle_event": "Валидация схемы при обработке события {{event}}.",

	"validation.socket.error.emit_unknown_event": "Данный тип {{event}} не существует в схеме валидации отправленных событий.",
	"validation.socket.error.handle_unknown_event": "Данный тип {{event}} не существует в схеме валидации полученных событий.",

	"validation.schema.error.required_user_id": "Уникальный идентификатор пользователя должен быть заполнен.",
	"validation.schema.error.required_socket_error": "Сообщение с ошибкой сокет-соединения должно быть заполнено.",
	"validation.schema.error.required_companion_id": "Уникальный идентификатор собеседника чата должен быть заполнен.",
	"validation.schema.error.required_message_id": "Уникальный идентификатор сообщения должен быть заполнен.",
	"validation.schema.error.required_chat_id": "Уникальный идентификатор чата должен быть заполнен.",
	"validation.schema.error.required_firstname": "Имя пользователя должно быть заполнено.",
	"validation.schema.error.required_thirdname": "Фамилия пользователя должна быть заполнена.",
	"validation.schema.error.incorrect_email": "Адрес электронной почты заполнен неверно.",
	"validation.schema.error.required_email": "Адрес электронной почты должен быть заполнен.",
	"validation.schema.error.required_phone": "Номер телефона должен быть заполнен.",
	"validation.schema.error.required_message_type": "Тип сообщения должен быть заполнен.",
	"validation.schema.error.required_message_create_date": "Дата создания сообщения должна быть заполнена корректно.",
	"validation.schema.error.required_message": "Тело сообщения должно быть заполнено.",
	"validation.schema.error.required_write_username": "Полное имя печатающего должно быть заполнено.",
	"validation.schema.error.required_friend_type": "Тип запроса дружбы должен быть заполнен.",
	"validation.schema.error.required_friend_id": "Уникальный идентификатор друга должен быть заполнен."
}
